src/bulk-load.js

```diff
@@ -115,6 +127,33 @@ module.exports = class BulkLoad extends EventEmitter {
     }
   }
 
+  /**
+   * @param {{checkConstraints : boolean , fireTriggers : boolean , keepNulls : boolean, tableLock: boolean}} bulkLoadOptions
```

Comment:
What do you think about changing this to be JSDoc compatible? Something like:

```js
/**
  @param {Object} bulkLoadOptions
  @param {boolean} bulkLoadOptions.checkConstraints - honors constraints during bulk load, it is disabled by default.
  @param {boolean} bulkLoadOptions.fireTriggers - honors insert triggers during bulk load, it is disabled by default.
  @param {boolean} bulkLoadOptions.keepNulls - honors null value passed, ignores the default values set on table.
  @param {boolean} bulkLoadOptions.tableLock- places a bulk update(BU) lock on table while performing bulk load. Uses row locks by default
*/
```

Reply:
Note: Not putting a `*` in front of each line of a multiline doc comment is my personal preference.

Reply:
Somewhere along the line I got confused between DocumentJS and JSDoc and ended up with that comment 😅

Reply:
No worries! I think in general, documentation.js is compatible with JSDoc, so we're good here now! 🎉